ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/util/BeaconStateUtil.java

```diff
@@ -622,6 +627,19 @@ public static Validator getValidatorByPubkey(BeaconState state, Bytes48 pubkey)
     return null;
   }
 
+  public static int getValidatorIndexByPubkey(BeaconState state, Bytes48 pubkey) {
```

Comment:
Would you mind adding javadoc comments for this method since it's a new public method?

Reply:
will do